contracts/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.dll
```

Comment:
Can I delete this folder if just for testing only?

Reply:
Now I moved "contracts" folder to AElf.Runtime.CSharp.Tests project. The reason why I copied the DLLs to a separate folder at build time was to prevent codecov injection into contract DLLs. When the DLLs from its output directory used, they are auto injected with codecov tracking methods and causing the test case to fail.